RELEASE.md

```diff
@@ -1,3 +1,34 @@
+# Release 2.2.1
```

Comment:
There's a few other PRs included in this rollup that seem worth mentioning:

## Features
- New logo used in the favicon (https://github.com/tensorflow/tensorboard/pull/3406)

## TensorBoard.dev updates: 
- The `tensorboard dev upload` command supports a `--plugins` option to explicitly indicate the desired plugins for which summary data should be uploaded (#3402, #3492)

## Bug fixes
- Eliminated sporadic HTTP 500 errors for XHRs that do markdown rendering (#3491)

Reply:
Done